### PR TITLE
feat: improve error message

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
## Description
This PR adds printing out instructions on reporting to GH discussions when UserError is thrown.

Error in terminal looks like this:

Snímek obrazovky 2023-08-07 v 7 24 37

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by martyndavies** [comment]
@Jakub-Vacek I realize now that the GH Discussions URL will be useless unless it's clickable. Is that possible to do with Oclif? If not, we should replace it with a short link.

**Comment by kysely** [comment]
At least in iTerm a link (or any path really) is clickable when holding Cmd. But I think it should be a shortlink anyways — it's better readable and we can keep it up to date.

**Comment by Jakub-Vacek** [comment]
Snímek obrazovky 2023-08-08 v 15 52 44

Short and highlighted link

**Comment by martyndavies** [comment]
@Jakub-Vacek Nice, looks good.

File: src/common/error.ts [hunk 1]
```diff
@@ -20,7 +22,24 @@ export const createUserError =
...
+
+    if (addSuffix) {
+      final =
+        final +
+        template(`\n\nIf you need help with this error, please raise an issue on GitHub Discussions: {underline.bold https://sfc.is/discussions}
+
+Please include the following information:
+command you ran
+error message
+version of the CLI, OS: superface cli/${VERSION} (${process.platform}-${process.arch}) ${process.release.name}-${process.version}
```
Comment (kysely) [inline src/common/error.ts:35]: nitpick: I'd add bullets here to make it visually more addressed

```suggestion
- command you ran
- error message
- version of the CLI, OS: superface cli/${VERSION} (${process.platform}-${process.arch}) ${process.release.name}-${process.version}
```

**Review by kysely — APPROVED** [review]
Looks okay, just left few nitpicks :)

**Pull request merged**